hyperSpec/vignettes/fileio.Rmd

```diff
@@ -1215,10 +1215,10 @@ more details for an already existing import filter.
 \index{PerkinElmer!Fluorescence}
 \index{PerkinElmer!ASCII long}
 
-The raw spectra of the `flu`{.r} data set (see also the respective vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
+The raw spectra of the `flu`{.r} data set (see also the separate vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
 
 We need a function that automatically reads all files specified by a pattern, such as `*.txt`{.r}.
-In order to gain speed, the spectra matrix should be preallocated after the first file is read.
+To gain speed, **hyperSpec** preallocates the spectra matrix after the first reading of the file.
```

Comment:
The meaning of corrected sentence is not the same. Is the new meaning really correct?